modules/core/shared/src/main/scala/lucuma/core/model/EmissionLine.scala

```diff
@@ -10,16 +10,14 @@ import coulomb.cats.implicits._
 import eu.timepit.refined.cats._
 import eu.timepit.refined.types.numeric.PosBigDecimal
 import lucuma.core.math.BrightnessUnits._
-import lucuma.core.math.Wavelength
 import lucuma.core.math.dimensional._
 import lucuma.core.math.units._
 import monocle.Focus
 import monocle.Lens
 
 final case class EmissionLine[T](
-  wavelength: Wavelength,
```

Comment:
So this is gone and instead it is in the map's index?

Reply:
Exactly